xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/buttons.less

```diff
@@ -183,4 +183,8 @@ input.button, .buttonwrapper button, .buttonwrapper a {
     color: @navbar-default-link-hover-color;
     background-color: @navbar-default-link-hover-bg;
   }
+}
+
+.btn-group > .btn:not(:first-child):not(:last-child):not(.dropdown-toggle) {
+  border-radius: 0 @border-radius-base @border-radius-base 0;
```

Comment:
To show the general idea, I'll update with a comment before removing the draft status.

Reply:
Actually, not a good selector, it was only working on a specific case but is not generic enough